config/routes.rb

```diff
@@ -232,8 +232,7 @@
           get 'add_grader_to_grouping'
           post 'csv_upload_grader_groups_mapping'
           post 'csv_upload_grader_criteria_mapping'
-          get 'download_grader_groupings_mapping'
-          get 'download_grader_criteria_mapping'
+          get 'download'
```

Comment:
Run the tests locally yourself. You'll need to update a bunch to use the new route.